waiter/src/waiter/core.clj

```diff
@@ -98,6 +98,7 @@
                      "app-name" :app-name-handler-fn
                      "apps" {"" :service-list-handler-fn
                              ["/" "instances"] :instances-list-handler-fn
+                             ["/" :service-id "/signal/" :signal-type "/" :instance-id] :signal-handler-fn
```

Comment:
Lexicographic order in this list. 
Also, please add a test for this entry in `test/waiter/core_test.clj#test-routes-mapper`.